{"data_type": "CVE", "data_format": "MITRE", "data_version": "4.0", "CVE_data_meta": {"ID": "CVE-2018-1269", "ASSIGNER": "cve@mitre.org"}, "affects": {"vendor": {"vendor_data": []}}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "CWE-388"}]}]}, "references": {"reference_data": [{"url": "https://www.cloudfoundry.org/blog/cve-2018-1269", "name": "https://www.cloudfoundry.org/blog/cve-2018-1269", "refsource": "CONFIRM", "tags": ["Third Party Advisory"]}]}, "description": {"description_data": [{"lang": "en", "value": "Cloud Foundry Loggregator, versions 89.x prior to 89.5 or 96.x prior to 96.1 or 99.x prior to 99.1 or 101.x prior to 101.9 or 102.x prior to 102.2, does not handle errors thrown while constructing certain http requests. A remote authenticated user may construct malicious requests to cause the traffic controller to leave dangling TCP connections, which could cause denial of service."}]}}
